src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -86,12 +86,31 @@ export class CalciteAlert {
   /** specify the scale of the button, defaults to m */
   @Prop({ reflect: true }) scale: Scale = "m";
 
+  @Prop() id: string;
```

Comment:
I don't think we want to add an id prop here. Can we use the element itself as a lookup in a map? See the tooltip manager for implementation details.

```
hoverTimeouts: WeakMap<HTMLCalciteTooltipElement, number> = new WeakMap();
```

```
if (hoverTimeouts.has(tooltip)) {
      window.clearTimeout(hoverTimeouts.get(tooltip));
      hoverTimeouts.delete(tooltip);
    }
```